_posts/en/newsletters/2024-05-15-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #302'
+permalink: /en/newsletters/2024/05/15/
+name: 2024-05-15-newsletter
+slug: 2024-05-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Release of utreexod beta:** Calvin Kim [posted][kim utreexo] to the
+  Bitcoin-Dev mailing list to announce the beta release of `utreexod`, a
+  full node with support for [utreexo][topic utreexo].  Utreexo allows a
+  node to store a small commitment to the state of the UTXO set rather
+  than the entire set itself; for example, a minimal commitment can be
+  32 bytes and the current full set is about 12 GB, making the
+  commitment on the order of about a billion times smaller.  To reduce
+  bandwidth, utreexo may store additional commitments, increasing its
+  use of disk space, but still keeping its chainstate roughly on the
+  order of a million times smaller than a traditional full node.
```

Comment:
```suggestion
  use of disk space, but still keeping its chainstate roughly
  a million times smaller than a traditional full node.
```

Reply:
Leaving as-is.  I think "roughly on the order of x" can span multiple magnitudes (e.g. 1 to 100), whereas "roughly x" probably means within a single magnitude (e.g. 1 to 3).  I'm being vague here because there are a lot of tradeoffs available with utreexo for bandwidth versus data storage.